src/triage/experiments/base.py

```diff
@@ -814,6 +839,126 @@ def _run_profile(self):
                          "in cProfile format.",
                          store)
 
+    def _fill_timechop_config_missing(self):
```

Comment:
I would probably put these defaults into a separate function in a separate module that can get imported here; it depends on the experiment config and a db engine but otherwise doesn't really need access to the instance. The Experiment base class is already pretty big and I think things that are logically separate from the running of an experiment (its output is the input of an Experiment!) make sense to remove.

Reply:
makes sense -- moved these out to a separate `defaults.py`